tools/eventkit-receiver/log.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	ui "github.com/elek/bubbles"
+	"strings"
+)
+
+type RawLog struct {
+	*ui.Text
+	repo  *Repo
+	start int
+}
+
+func NewRawLog(repo *Repo) *RawLog {
+	r := &RawLog{
+		repo: repo,
+	}
+	r.Text = ui.NewText(r.render)
+	return r
+}
+
+func (l *RawLog) render() string {
+	out := ""
+	if l.Text.GetHeight() > 5 {
+		for ; l.start+l.Text.GetHeight() <= len(l.repo.Events); l.start = l.start + l.Text.GetHeight() - 5 {
```

Comment:
Is this intentional? I would put the condition inside the loop and avoid the first empty section.

```
for l.start+l.Text.GetHeight() <= len(l.repo.Events) {
    l.start = l.start + l.Text.GetHeight() - 5
}
```